src/platform/utilities/api/index.js

```diff
@@ -56,22 +56,14 @@ export function apiRequest(resource, optionalSettings = {}, success, error) {
         ? response.json()
         : Promise.resolve(response);
 
-      if (!response.ok) {
-        const { pathname } = window.location;
-        const shouldRedirectToLogin =
-          response.status === 401 && !pathname.includes('auth/login/callback');
-
-        if (shouldRedirectToLogin) {
-          const loginUrl = appendQuery(environment.BASE_URL, {
-            next: pathname,
-          });
-          window.location.href = loginUrl;
-        }
```

Comment:
I've removed this as it seems that any authenticated requests are made from components that already handle the logged out scenario. The same redirect occurs for any apps that use `RequiredLoginView`. Ultimately, it doesn't seem like the responsibility of redirecting and clearing the session data belongs to this method. We should keep it lean and delegate that responsibility to the components.